### DISCUSSION TITLE
Why does `fs_extra::file::move_file`  use `fs_extra::file::copy` instead of `std::fs::rename`

### DISCUSSION THREAD

**Discussion by ghost** [webdesus/fs_extra#56] [Q&A] [answered]
In this function:

https://github.com/webdesus/fs_extra/blob/752fc4d1e6e4eff3e506bffe7196e2a537c6c52b/src/file.rs#L236

Why does `fs_extra::file::move_file`  use `fs_extra::file::copy` instead of `std::fs::rename` ?

If I need to move a folder with many large files, `fs_extra::file::copy` will be much slower than `std::fs::rename`...

**Comment by webdesus** [comment, marked as answer]
Hi @CrazyPandas !
Thank you for you question. 
I begin again work on a lib and we have plan to fix it.  More details here #20

**Reply by ghost** [reply]
That's great news. Thank you for your work.
